forge-gui/res/cardsfolder/upcoming/hardy_outlander.txt

```diff
@@ -2,10 +2,8 @@ Name:Hardy Outlander
 ManaCost:1 G
 Types:Legendary Enchantment Background
 S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddTrigger$ AttacksPlayer | Description$ Commander creatures you own have "Whenever this creature attacks a player, if no opponent has more life than that player, another target creature you control gets +X/+X until end of turn, where X is this creature's power."
-SVar:AttacksPlayer:Mode$ Attacks | ValidCard$ Card.Self | Attacked$ Player | Execute$ TrigPump | CheckSVar$ Y | SVarCompare$ GEZ | TriggerDescription$ Whenever this creature attacks a player, if no opponent has more life than that player, another target creature you control gets +X/+X until end of turn, where X is this creature's power.
+SVar:AttacksPlayer:Mode$ Attacks | ValidCard$ Card.Self | Attacked$ Player | Execute$ TrigPump | Condition$ AttackedPlayerWithMostLife | TriggerDescription$ Whenever this creature attacks a player, if no opponent has more life than that player, another target creature you control gets +X/+X until end of turn, where X is this creature's power.
```

Comment:
Not sure this cleanup works

the other card that uses it cares about all player, but this one should ignore Allies?

Reply:
Good catch – I moved this to #584